src/big/big_decimal.cr

```diff
@@ -260,7 +260,7 @@ struct BigDecimal < Number
     in_scale(new_scale.scale)
   end
 
-  private def in_scale(new_scale : UInt64) : BigDecimal
+  def in_scale(new_scale : UInt64) : BigDecimal
```

Comment:
This method should probably not be public API but `protected` (or `:nodoc:`).

Reply:
I don't see any reason to make it `protected`, why not bring it into the public API?

Reply:
I can't tell whether it would be functionally relevant for the public API. But there's already `#scale_to` which seems to be intended as the public facing front. 

If it goes public, it should be documented.

Reply:
I've made it `:nodoc:`. `#scale_to` has a bit odd (imo) interface forcing you to pass another `BigDecimal` - which is not always wanted if you want to just scale a number at hand.

Reply:
Feel free to open a separate issue about changing the API! But this PR is better focused on fixing the bug.